### ISSUE TITLE
Not able to set ComposeUIViewController background to transparent.

### ISSUE THREAD

**Issue by khalid64927** [JetBrains/compose-multiplatform#4108]
**Describe the bug**
I tried to set background color for ComposeUIViewController in [iOS code in this demo repo](https://github.com/khalid64927/kmp-tes-app/blob/main/iosApp/iosApp/ComposeView.swift#22) but its not reflecting its always white.
This probably because the White BG color is set in [ComposeContainer](https://github.com/JetBrains/compose-multiplatform-core/blob/jb-main/compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeContainer.uikit.kt#L147)

**Affected platforms**
Select one of the platforms below:
- iOS

**Versions**
- Kotlin version*:  1.9.21
- Compose Multiplatform version*:  1.5.11
- OS version(s)* (required for Desktop and iOS issues):  Tested in iOS Simulator iPhone 12 Pro running iOS 15.2
- OS architecture (x86 or arm64): arm64

**To Reproduce**
Steps and/or the code snippet to reproduce the behavior:
1. Checkout [this repo ](https://github.com/khalid64927/kmp-tes-app)and run iOS or simply try to set MainViewController Background to transparent in Jetbrains compose template
2. After making changes run iOS app
3. You will see iOS App BG color as white 

**Expected behavior**
BG color should be transparent.

**Screenshots**
Observed behaviour 
![simulator_screenshot_B96D1EEC-0E21-456B-BEA5-3DCDB59271D8](https://github.com/JetBrains/compose-multiplatform/assets/2426309/5adf95d5-7222-4f33-89d9-d7180b527911)

**Video**
https://github.com/JetBrains/compose-multiplatform/assets/2426309/d97e97c1-c95b-4404-8e7e-92569b071958

**Additional context**
I am working on developing a Payment SDK using Compose Multiplatform. Essentially, this SDK involves creating a BottomSheet that pops up to display payment options on top of an existing screen. The existing screen could be either a React Native App or a Native App. It is crucial that when BottomSheet is visible the background RN or Native app screen remains visible, and for this reason, the ComposeUIViewController needs to be transparent.

**Comment by khalid64927** [comment]
I tried to add shared module on to React Native project
https://github.com/JetBrains/compose-multiplatform/assets/2426309/d97e97c1-c95b-4404-8e7e-92569b071958

**Comment by MatkovIvan** [comment]
This problem was [already discussed](https://kotlinlang.slack.com/archives/C0346LWVBJ4/p1685005635951389), but I don't see a separate issue for this topic on GitHub, so let's track it here.

This problem is internally related to how interop views work, so it makes sense to mention this issue: https://github.com/JetBrains/compose-multiplatform/issues/3154
However, it seems that it can be solved independently of interop transparency. I'll discuss this with the team, no promises about ETA for now.

**Comment by khalid64927** [comment]
Thanks, the good folks at discussion thread implemented a fix which is working fine for current compose version (1.5.11).

**Comment by chrisbanes** [comment]
Inspired by the Slack discussion linked above, I've come up with a workaround which is far less invasive: https://gist.github.com/chrisbanes/bdcf1d5d79c517020d5931524d0f3523.

Just call  `forceTransparentBackground()` from your Compose content.

**Comment by MatkovIvan** [comment]
Fixed in https://github.com/JetBrains/compose-multiplatform-core/pull/1022

**Comment by kocheick** [comment]
I'm using compose version 1.6.1
I can't seem to find any opaque variable, the composeUIViewController only has a delegate variable, what am I missing here ?

**Comment by elijah-semyonov** [comment]
@kocheick 
It's inside `ComposeUIViewControllerConfiguration`, you can modify its properties in `configure` closure of `ComposeUIViewController`

**Comment by MatkovIvan** [comment]
> I'm using compose version 1.6.1

There is no such version for Compose Multiplatform yet

**Comment by kocheick** [comment]
> > I'm using compose version 1.6.1
> 
> There is no such version for Compose Multiplatform yet

Thanks for looking into it, I was referring to the Compose Runtime version as 1.6.1 


> I'm using compose version 1.6.1 I can't seem to find any opaque variable, the composeUIViewController only has a delegate variable, what am I missing here ?

 I actually had to update the Compose Plugin in order to see get new update
```

#compose-plugin = "1.5.12".   // went from this to the below version 
compose-plugin = "1.6.0-rc01"
```

The fix is not available with the plugin version 1.5.12 or below. Now the png rendered in compose gets no white background by default unless specified. Thanks

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-4108">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
